src/hooks/useProductById.js

```diff
@@ -1,7 +1,12 @@
 import { useAPIData } from 'utils/hooks/useAPIData';
 
-const useProductById = (productId) => {
-    const queries = [`:d = at(document.id, "${productId}")`];
+//retrieve multiple products by their id's
+//https://wizeline-academy.cdn.prismic.io/api/v2/documents/search?ref=YZaBvBIAACgAvnOP&q=[[any%28document.id%2C+%5B%22YZWdwRIAACkAumb-%22%2C+%22YZWlSRIAACoAuoj9%22%5D%29]]
+//TODO: extend this function to retrieve multiple products
+const useProductById = (productIds = [], pageSize) => {
+
+    const concatenatedProductIds = productIds.map((productId)=> `"${productId}"`).join(',');
+    const queries = [`any(document.id, [${concatenatedProductIds}])`];
     const mapFunction = (
```

Comment:
If you move this on to a  utils file that only stores the fn would be awesome to have the file as clear as possible